integ-test/src/test/java/org/opensearch/sql/legacy/PrettyFormatResponseIT.java

```diff
@@ -483,6 +484,7 @@ public void joinQuerySelectOnlyOnOneTable() throws Exception {
     assertContainsData(getDataRows(response), fields);
   }
 
+  @Disabled("Disabled temporarily due to JSON format incompatibility with V2 and Legacy")
```

Comment:
what do you mean by this?  Is this test case now supported in V2 and it fails?

Reply:
This may have been a mistake. I think this was failing due to the data type refactor at first, and when I asked about it in standup JSON format was the reason that was suggested for why it started failing all of a sudden, so I just put that as the description. This has been fixed since then, so I will remove these annotations.

Reply:
https://github.com/Bit-Quill/opensearch-project-sql/pull/258/commits/2ccebd9d363ec4f43b4b8eec3c30a4f9218c0653